build.sbt

```diff
@@ -1,55 +1,21 @@
 import ReleaseTransformations._
+import sbtversionpolicy.withsbtrelease.ReleaseVersion
 
 name := "simple-configuration"
 
-ThisBuild / scmInfo := Some(
-  ScmInfo(
-    url("https://github.com/guardian/simple-configuration"),
-    "scm:git@github.com:guardian/simple-configuration.git"
-  )
-)
-
-ThisBuild / homepage := Some(url("https://github.com/guardian/simple-configuration"))
-
-ThisBuild / developers := List(Developer(
-  id = "Guardian",
-  name = "Guardian",
-  email = null,
-  url = url("https://github.com/guardian")
-))
-
 val scala_2_12: String = "2.12.19"
 val scala_2_13: String = "2.13.13"
 
 val awsSdkVersion = "2.23.21"
 
 scalaVersion := scala_2_13
 
-ThisBuild / publishTo := sonatypePublishTo.value
-
 val sharedSettings = Seq(
   scalaVersion := scala_2_13,
   crossScalaVersions := Seq(scala_2_12, scala_2_13),
-  releaseCrossBuild := true,
-  licenses += ("Apache-2.0", url(
-    "http://www.apache.org/licenses/LICENSE-2.0.html"
-  )),
+  licenses := Seq(License.Apache2),
   organization := "com.gu",
-  releasePublishArtifactsAction := PgpKeys.publishSigned.value,
-  releaseProcess := Seq[ReleaseStep](
-    checkSnapshotDependencies,
-    inquireVersions,
-    runClean,
-    runTest,
-    setReleaseVersion,
-    commitReleaseVersion,
-    tagRelease,
-    releaseStepCommandAndRemaining("+publishSigned"),
-    releaseStepCommand("sonatypeBundleRelease"),
-    setNextVersion,
-    commitNextVersion,
-    pushChanges
-  )
+  scalacOptions := Seq("-release:11")
```

Comment:
I like this, but do be aware that this will force any project using this library to upgrade to Java 11 ([not a bad thing!](https://docs.google.com/document/d/1ZR-YnaXCT5_gLVmTCeGs0mWd3KPaAozPjQK8uUzHZ9w/edit?usp=sharing)).

We should be on Java 11 everywhere, but `simple-configuration` is used [in a lot of places](https://github.com/search?q=org%3Aguardian%20%22simple-configuration%22&type=code) and it's possible some of them are still on Java 8! They really should upgrade...